_config/middleware.yml

```diff
@@ -0,0 +1,8 @@
+---
+Name: subsitesmiddleware
```

Comment:
Can you make sure this is initialised after the session middleware?

Reply:
Updated, thanks